.github/workflows/meshery-cloud-release.yml

```diff
@@ -112,7 +112,7 @@ jobs:
           cd meshery-cloud
           sudo git pull
           sudo docker pull layer5/meshery-cloud:staging-latest
-          sudo make stg-deploy
+          sudo make docker-stg-deploy
```

Comment:
@MUzairS15 @theBeginner86 are we still wanting this change? I'm fine with it.

I would like to see a deployment step included for cloud.layer5.io...

Reply:
This is a new make target `docker-prod-<env name>` which uses compose app instead of bunch of docker commands